libs/dao-data/src/Profile.ts

```diff
@@ -66,3 +77,59 @@ export default class Profile {
     return await provider.lookupAddress(address);
   }
 }
+
+const profilModel = {
```

Comment:
Do we need this here?

Reply:
We can remove it. It has the metadata of for the hash we use above. I don't mind either way